pkg/client/client_test.go

```diff
@@ -88,6 +88,8 @@ func TestClient(t *testing.T) {
 			fp = fmt.Sprintf("%s/%s", dataDir, "resp.show.ip.bgp.summary.vrf.all.1.json")
 		case "show interface transceiver details":
 			fp = fmt.Sprintf("%s/%s", dataDir, "resp.show.interface.transceiver.details.1.json")
+		case "show clock":
```

Comment:
@tao-sj , in your initial PR I mentioned the `supportedCommand` map. Please add it here. That is the only change here. Otherwise, LGMT 👍

Reply:
thanks! updated